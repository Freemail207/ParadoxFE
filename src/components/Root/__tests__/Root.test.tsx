import React from 'react';
import { shallow } from 'enzyme';
import Root from '../Root'

describe('Root Component Test', () => {
    test('Check Root Component render', async () => {
        const component = shallow(<Root />);

        expect(component).toMatchSnapshot();
    });
});
