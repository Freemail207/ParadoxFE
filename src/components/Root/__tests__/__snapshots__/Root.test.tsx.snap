// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root Component Test Check Root Component render 1`] = `ShallowWrapper {}`;
